.gitlab-ci.yml

```diff
@@ -122,6 +123,69 @@ before_script:
     ## Advanced DNS checks
     - ansible-playbook -i inventory/inventory.ini -e ansible_python_interpreter=${PYPATH} -u $SSH_USER -e ansible_ssh_user=$SSH_USER $SSH_ARGS -b --become-user=root tests/testcases/040_check-network-adv.yml $LOG_LEVEL
 
+    ## Idempotency checks 1/5 (repeat deployment)
+    - >
+      if [ "${IDEMPOT_CHECK}" = "true" ]; then
+        ansible-playbook -i inventory/inventory.ini -u $SSH_USER -e ansible_ssh_user=$SSH_USER $SSH_ARGS
+        -b --become-user=root -e cloud_provider=gce $LOG_LEVEL -e kube_network_plugin=${KUBE_NETWORK_PLUGIN}
+        --private-key=${HOME}/.ssh/id_rsa
+        -e bootstrap_os=${BOOTSTRAP_OS}
+        -e ansible_python_interpreter=${PYPATH}
+        -e download_run_once=true
+        -e download_localhost=true
+        -e deploy_netchecker=true
+        -e resolvconf_mode=${RESOLVCONF_MODE}
+        -e local_release_dir=${PWD}/downloads
+        -e etcd_deployment_type=${ETCD_DEPLOYMENT}
+        -e kubelet_deployment_type=${KUBELET_DEPLOYMENT}
+        cluster.yml;
+      fi
```

Comment:
Let's start by breaking this down to config sets that are stored in vars at least. ideally in tests dir in yaml format which we can source. Definitely not in scope of this PR, but we can start thinking now